lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -98,8 +98,14 @@ class ComicFormViewModel {
         self.issue.value = comic.issueNumber ?? ""
         self.publisher.value = comic.publisher ?? ""
         self.release.value = comic.releaseYear ?? ""
-        self.genre.value = Comic.Genre.convertStringToGenre(genre: comic.genre)
-        self.condition.value = Comic.Condition.convertStringToCondition(condition: comic.condition)
+        self.genre.value = Genre.convertStringToGenre(genre: comic.genre)
+        self.condition.value = Condition.convertStringToCondition(condition: comic.condition)
+    }
+
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Avertical_whitespace)